libs/infra/images.py

```diff
@@ -19,6 +19,20 @@ class Cirros:
     DEFAULT_MEMORY_SIZE: str = "64M"
 
 
+@dataclass
+class Alpine:
+    RAW_IMG: str | None = None
+    RAW_IMG_GZ: str | None = None
+    RAW_IMG_XZ: str | None = None
+    QCOW2_IMG: str | None = None
+    QCOW2_IMG_GZ: str | None = None
+    QCOW2_IMG_XZ: str | None = None
+    DISK_DEMO: str | None = None
```

Comment:
With this PR, you only add the `QCOW2_IMG`, right? So we can remove the rest for now?

```suggestion
    QCOW2_IMG: str | None = None
```

Reply:
sure